src/lkl/syscall-overrides.c

```diff
@@ -1,6 +1,10 @@
-#include "lkl/syscall-overrides-futex.h"
+#include <sys/syscall.h>
+#include "lkl/syscall-overrides-fstat.h"
 #include "lkl/posix-host.h"
 
 void register_lkl_syscall_overrides()
 {
+    orig_fstat = lkl_replace_syscall(__lkl__NR_fstat, syscall_fstat_override);
```

Comment:
If you move this into a `sgxlkl_register_stat_overrides` function in `syscall-overrides-fstat.c` then the two original call functions can be static.

Reply:
Good point. Will do.